### DISCUSSION TITLE
Ability to mute specific sounds effects

### DISCUSSION THREAD

**Discussion by ecnepsnai** [OpenRCT2/OpenRCT2#15664] [Feature requests] [answered]
From this reddit comment: https://www.reddit.com/r/rct/comments/3yq32k/openrct2_feature_highlights_fps_counter_message/cyfxhbm

> [...] Now if only we could do something about those deafening restrooms

We should add the ability to mute specific building sounds from the sounds option window.

**Comment by Broxzier** [comment]
Compiling a list of all sounds beforehand could be helpful. Here are some of the sounds I am thinking off:

**Shops / Stalls**
- Cash
- Toilets

**Rides**
- Cash
- Movement
- Chainlift
- Block breaks
- Doorways
- Vehicle specific noise (Trains, trams)
- Freefall launch
- Circus
- Crash

**Peeps**
- Crowd
- Cheer
- Vomit

**Game**
- Message
- Chat
- Rain

**Comment by telk5093** [comment]
I think it is good idea to add additional ones to Rides category of @Broxzier: Merry-go-round and Dodgem BGM

**Comment by Gymnasiast** [comment]
That's already possible by disabling music, either entirely (Options window) or per ride.

**Comment by telk5093** [comment]
Yeah, you're right. I forgot it.

**Comment by IntelOrca** [comment]
Yes, the only ride music you can't disable is Circus.

**Comment by karst** [comment, marked as answer]
This is now possible using asset packs. It is not quite as convenient as an advanced UI, but the ability is there.

**Comment by J-PIE-314** [comment]
This looks like something that should be implemented as a plug-in.
